source/includes/_intro.md

```diff
@@ -394,25 +394,267 @@ The first step in most situations will be to instantiate a client. (TODO these c
+and a £5.00 surcharge. Any dispatch methods that incur an additional cost
+should also be listed separately. A total price should be displayed in a
+highlighted font showing the price the customer will pay (by adding the prices
+together).
+
+### Display special seat conditions to customer before purchase
+Some seats in venues may have restricted views of the stage, or other
+information associated with them. It's important that the customers are made
+aware of any special seat conditions that will apply before they purchase.
+
+**Representative Test:** User confirms reservation of 3 seats in CIRCLE-B for
+event 7AB including seats F1-F3<br/>
+When the reservation is made<br/>
+Then the confirmation screen should show that these seats have restricted
+views, and the seat text for F2.
+
+### Release seats if customer does not proceed with the booking flow
+A customer may change their mind about the number or type of seats to be
+reserved for a performance. If your customer selects and reserves seats, then
+goes back to select and reserve additional seats or removes the reserved seats
+from their basket, the initial seats should be [released](#release).
+
+Failure to release the initial reservation will mean the customer is unable to
+reserve seats (even though they are actually available). Ingresso will
+automatically release any reserve after 15 minutes of inactivity, but you must
+release seats as soon as possible. Ingresso reserves the right to disable the
+API account of any partner that does not release tickets when they should be.
+
+**Representative Test:** User makes a reservation for any ticket then clicks
+"back" or otherwise moves away from the reservation confirmation screen<br/>
+When the customer moves away from the reservation screen<br/>
+Then your application should call the `release` resource of the
+Ingresso API. This must be done before any further call to `reserve`.<br/>
+Optionally it should also display a message to the customer indicating the
+seat reservation has been released.
+
+### Display friendly error messages during purchase
+When purchasing on the Internet, it can be very disconcerting for customers
+when errors occur. Make sure any errors are shown to the customer, with a
+message that their card has not been charged.
+
+**Representative Test:** User makes a reservation for any ticket type on event 7AB, and
+at purchase enters a postcode of `W5 4JJ`.<br/>
+When the customer enters a postcode of `W5 4JJ` in the address details of the
+purchase confirmation,<br/>
+Then the Ingresso API will generate a booking failure and return the
+result. The application should show the customer that an error occurred, the
+reservation was not successful and that their card has not been charged.
+
+### Display different send methods including eTickets
+Various venues support different methods of sending tickets (including post,
+pickup from box office, or printable eTickets). Some of these may carry an
```

Comment:
I would use "collect" rather than "pickup" just because that is more common terminology in the industry, a very minor point...